gammapy/spectrum/core.py

```diff
@@ -427,6 +396,20 @@ def stats(self, nbin):
             livetime = self.exposure,
         )
         return ObservationStats(**kwargs)
+    
+    def predicted_counts(self, model):
```

Comment:
Document what this method returns, i.e. a `CountSpectrum`.

Rename to `n_pred`?